### PR TITLE
Added SSO

### PR DESCRIPTION
**Pull request by avasquez614**
Improved logging for Alfresco repo.

### CODE REVIEW

File: src/main/java/org/craftercms/studio/impl/v1/repository/alfresco/AlfrescoContentRepository.java [hunk 1]
```diff
@@ -278,106 +293,141 @@ protected boolean copyContentInternal(String fromPath, String toPath, String new
...
         long duration = System.currentTimeMillis() - startTime;
-        logger.debug("TRACE: alfrescoMultipartPostRequest(String uri, Map<String, String> params, InputStream body, String bodyMimeType, String charSet); {0}, {1}, {2}, {3}, {4}\n\t\tDuration: {5}", uri, params, "body", bodyMimeType, charSet, duration);
+        logger.debug("alfrescoPostRequest(uri = {0}, params = {1}, body = {2}, bodyMimeType = {3}) ({4} ms)", uri,
+                     params, "stream", bodyMimeType, duration);
         InputStream responseStream = postMethod.getResponseBodyAsStream();
         String response = IOUtils.toString(responseStream);
         return response;
     }
 
+//    /**
```
Comment (russdanner) [inline src/main/java/org/craftercms/studio/impl/v1/repository/alfresco/AlfrescoContentRepository.java]: Please do not comment out code.

File: src/main/java/org/craftercms/studio/impl/v1/util/DomUtils.java [hunk 2]
```diff
@@ -1,36 +1,38 @@
 /*******************************************************************************
```
Comment (russdanner) [inline src/main/java/org/craftercms/studio/impl/v1/util/DomUtils.java:1]: Why this file?

File: src/main/java/org/craftercms/studio/impl/v1/repository/alfresco/AlfrescoContentRepository.java (hunk 1 above)
Comment (avasquez614) [inline src/main/java/org/craftercms/studio/impl/v1/repository/alfresco/AlfrescoContentRepository.java]: Ok I will remove the commented code.

File: src/main/java/org/craftercms/studio/impl/v1/util/DomUtils.java (hunk 2 above)
Comment (avasquez614) [inline src/main/java/org/craftercms/studio/impl/v1/util/DomUtils.java:1]: I just indented it. The old indentation seemed pretty weird.

**Pull request merged**